tests/FSharp.Test.Utilities/ProjectGeneration.fs

```diff
@@ -1027,11 +1028,12 @@ type ProjectWorkflowBuilder
 
     member this.Execute(workflow: Async<WorkflowContext>) =
         try
-            Async.RunSynchronously(workflow, timeout = defaultArg runTimeout 600_000)
+            // We don't want the defaultCancellationToken.
```

Comment:
For reference, it would be nice to specify the reason why :)

Reply:
actually let's revert this one, this shouldn't be a problem anymore with the tests doing `CancelDefaultToken` contained.